operator-pipeline-images/Dockerfile

```diff
@@ -21,9 +21,13 @@ RUN dnf update -y && \
     dnf install -y \
     findutils \
     git \
+    wget \
```

Comment:
The original dockerfile should not be updated at all. Can you revert the changes.